CONTRIBUTING.md

```diff
@@ -20,102 +20,103 @@ The code of conduct for cactbot can be found here:
 
 ## Issues and Bug Reports
 
-Please file all issues with cactbot on github,
-via this url:
+Please file all issues with cactbot on github, via this url:
 [https://github.com/quisquous/cactbot/issues/new](https://github.com/quisquous/cactbot/issues/new)
 
 ## Pull Requests
 
-It's vastly preferable to make a lot of small pull requests (that are easy to review)
-than very large pull requests (that may conflict or take a while to land).
-Each pull request should be small and be self-contained in terms of what it is changing.
+It's vastly preferable to make a lot of small pull requests (that are easy to
+review) than very large pull requests (that may conflict or take a while to
+land). Each pull request should be small and be self-contained in terms of what
+it is changing.
 
-If your change is small, just send a pull request and we can have a conversation there.
-If your change is big, consider having a conversation before you embark on a lot of work.
+If your change is small, just send a pull request and we can have a conversation
```

Comment:
I think this reformatting here is not great in terms of removing  semantic linebreaks.

How hard would it be to add linting without line limits? (Or to increase the line limit to like 100 such that these lines don't get reformatted?) \*<_<*

Reply:
The level of effort isn't too high; the issue that I have is that to override the configuration we'd have to create a configuration file, and we can't use package.json. The ruby implementation lets you define your linting rules inline with the command-line tool, but the same does not exist for the Node version.

For example: https://github.com/eslint/eslint/blob/master/.markdownlint.yml

I don't have a huge issue with adding another file, but it's a little ugh in my opinion. Other than that, I have no issues; let me know what you think!

Reply:
An additional config file seems fine.  It's not clear that there's really another option.